lib/vyasa_web/components/core_components.ex

```diff
@@ -779,19 +779,30 @@ defmodule VyasaWeb.CoreComponents do
 
       <.back navigate={~p"/posts"}>Back to posts</.back>
   """
-  attr :navigate, :any, required: true
+  attr :navigate, :any, default: nil
+  attr :patch, :any, default: nil
   slot :inner_block, required: true
 
   def back(assigns) do
     ~H"""
     <div class="mt-16">
-      <.link
-        navigate={@navigate}
-        class="text-sm font-semibold leading-6 text-zinc-900 hover:text-zinc-700"
-      >
-        <.icon name="hero-arrow-left-solid" class="h-3 w-3" />
-        <%= render_slot(@inner_block) %>
-      </.link>
+      <%= if @patch do %>
```

Comment:
this gives preference to `@patch` if both are provided.
doing so will always attempt to avoid a full-page reload and the existing socket doesn't get killed

Reply:
patch >> navigate for this explorer style